src/components/about-panel/index.js

```diff
@@ -13,17 +18,37 @@ const { shell, remote } = window.require( 'electron' );
 
 class AboutPanel extends Component {
 	render() {
+		const repositoryLink = (
+			<a href="#" onClick={ () => shell.openExternal( 'https://github.com/pento/testpress' ) }>
```

Comment:
Now that we have `Button` available, this could now be a `Button` styled as a link, I think?

Reply:
👌 9b98d61